content/docs/components-and-props.md

```diff
@@ -16,23 +16,27 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+Komponen mempermudah anda untuk memecah UI menjadi bagian tersendiri, bagian yang bisa digunakan kembali, dan berpikir tentang setiap potongan dalam isolasi.
```

Comment:
UI -> antarmuka pengguna

This line is missing:
`You can find a [detailed component API reference here](/docs/react-component.html).`

😄